visualizer/style/exampleGenerationLinks.css

```diff
@@ -0,0 +1,61 @@
+.currentExampleLink.coveredExampleLink {
+  text-decoration: underline;
+  color: #268BD2;
+  cursor: pointer;
+  font-weight: bold;
+}
+
+.coveredExampleLink {
+  text-decoration: underline;
+  /*color: hsl(0, 0%, 50%);*/
```

Comment:
It's best not to check in commented-out code. If you have a good reason to, it should have a comment explaining why -- otherwise this stuff tends to hang around forever and nobody remembers why :-)